Gemfile

```
@@ -5,6 +5,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.5'
 gem 'bootsnap', require: false
+gem 'devise_token_auth', '~> 1.2', '>= 1.2.1'
```

I think this is the same
```suggestion
gem 'devise_token_auth', '~> 1.2.1'
```